templates/results.html

```diff
@@ -0,0 +1,87 @@
+{% extends "base.html" %}
+
+{% block title %} Results {% endblock %}
```

Comment:
To make the title of the page actually based on the `subject` rather than the generic "Result":
```suggestion
{% block title %} {{ subject|replace('\n', ' ')|safe }} {% endblock %}
```